apps/web/modules/videos/views/videos-single-view.tsx

```diff
@@ -103,10 +103,10 @@ export default function JoinCall(props: PageProps) {
           <CalAiTranscribe />
         </div>
         <div style={{ zIndex: 2, position: "relative" }}>
-          {booking?.user?.organization?.calVideoLogo ? (
+          {calVideoLogo ? (
             <img
               className="min-w-16 min-h-16 fixed z-10 hidden aspect-square h-16 w-16 rounded-full sm:inline-block"
-              src={booking.user.organization.calVideoLogo}
```

Comment:
As said before. Why not an URL?

Reply:
This is just a temporary fix till we have calVideoUrl in DB (like avatarUrl)